### ISSUE TITLE
stream read group when block 0

### ISSUE THREAD

**Issue by lenonqing** [rueian/rueidis#77]
read group when block 0, will return `context.DeadlineExceeded`

```go
xReadGroup = c.XReadGroup(context.Background(), XReadGroupArgs{
	Group:    "group",
	Consumer: "consumer",
	Streams:  []string{key, ">"},
	Block:    0,
})
fmt.Println(xReadGroup.Err())
```

Output:
`
context deadline exceeded
`

**Comment by rueian** [comment]
Hi @lenonqing,

Thank you for reporting this. This could take a couple of days.

**Comment by rueian** [comment]
Hi @lenonqing, this should be fix in v0.0.70.

**Comment by lenonqing** [comment]
ok,thanks.
